src/libraries/System.Private.CoreLib/src/System/Runtime/GCFrameRegistration.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Loader;
+
+namespace System.Runtime
+{
+#pragma warning disable 0414, IDE0044
```

Comment:
```suggestion
        [StructLayout(LayoutKind.Sequential)]
```
This is better way to avoid these warnings.

Reply:
Done